### ISSUE TITLE
Suggestion: course release date

### ISSUE THREAD

**Issue by alina-artcibasova** [koaning/calmcode-feedback#154]
Hello! Thank you very much for your site, I'm enjoying watching your courses a lot! However, one thing I keep wishing for is having some sort of a timestamp for the courses indicating when were they initially published and maybe a dashboard sorting the courses accordingly.

For example with scikit learn course - Boston dataset controversy is referenced, but it's unclear how recent it is just from the videos, and one has to follow external links to find out.

Another example is the python package setup tools. You recommend joint usage of vs code and jupiter notebooks, but currently vs code has a rather robust support for jupiter notebooks as well, so it's hard to tell whether it's down to personal preference, or at the time of filming it was simply unavailable.

I understand that this may be a lot of work to implement, but in case it is something which can be easily automated I figured I would bring it up.

Thanks!

**Comment by koaning** [comment]
This sounds like a fair suggestion. 

One part that's a bit annoying is that each lesson might get updated. It's happened quite often that after a year we updated a video. This could get reflected in some sort of a "most recent edit date" kind of a mention at the bottom of the appendix. Would that suffice?

**Comment by alina-artcibasova** [comment]
Yeah, I think that's totally fair. Might also be helpful for people who already watched the course, prompt them to check out what has changed. Thank you!

**Comment by koaning** [comment]
Will explore how easy it might be to add that. I'm hoping to automate it by looking at the Github commits. It's on the "to be explored"-list!
